modules/ROOT/partials/configuration/highlight_on_focus.adoc

```diff
@@ -0,0 +1,34 @@
+[[highlight_on_focus]]
+== `+highlight_on_focus+`
+
+The `+highlight_on_focus+` option adds a blue outline to an instantiated {productname} editor when that editor is made the input focus.
+
+This allows users to clearly see when the editor is in focus, or which editor has focus if more than one {productname} instance is available.
+
+*Type:* `+Boolean+`
+
+*Default value:* `+false+`
+
+*Possible values:* `+true+`, `+false+`
+
+=== Example: using `+hidden_input+`
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",
+  highlight_on_focus: true
+});
+----
+
+=== Using `+highlight_on_focus+` with custom skins
+
+If a xref:creating-a-skin.adoc[custom skin] is being used, two at-rules must be updated for the `+highlight_on_focus+` option to work correctly.
+
+The two rules and their required updates are:
+
+[source,css]
+----
+@edit-area-border-color: #2D6ADF;
+@edit-area-border-width: 2px;
+----
```

Comment:
```suggestion
If a xref:creating-a-skin.adoc[custom skin] is being used, two variables can be updated for customized appearance of the `+highlight_on_focus+` option.

The two rules and their default values are:

[source,css]
----
@edit-area-border-color: #2D6ADF;
@edit-area-border-width: 2px;
----
```
These are variables in the less files, not at-rules used in css. They can be customized, but if a user has a custom skin this will work as normal with the default colour/width.